db/schema.rb

```diff
@@ -15,6 +15,19 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "carriers", force: :cascade do |t|
+    t.integer "item_id"
```

Comment:
This needs to be a `string` type because the IDs they generate include text characters, i.e. (B10A0901)

Reply:
ids are now strings 👍